addons/website_sale/models/website_snippet_filter.py

```diff
@@ -87,20 +87,25 @@ def _filter_records_to_values(self, records, is_sample=False):
         return res_products
 
     @api.model
-    def _get_products(self, mode, **kwargs):
+    def _get_products(self, mode, search_domain=None, **kwargs):
```

Comment:
No need for argument we are already getting domain from context [see here](https://github.com/odoo/odoo/pull/166105/files#diff-981dfdc95cf648a923ae7e48c50dfb6cd2d5b24efaa6e56f22e46298e6ecfd1fR94)
```suggestion
    def _get_products(self, mode, **kwargs):
```

Reply:
Done the changes :+1: 
Thankyou